### ISSUE TITLE
-Wmissing-braces in drivers/net/geneve.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#267]
```
drivers/net/geneve.c:428:29: error: suggest braces around initialization of subobject [-Werror,-Wmissing-braces]
                struct in6_addr addr6 = { 0 };
                                          ^
                                          {}
1 error generated.
```

Patch sent: https://lore.kernel.org/lkml/20181112221248.11477-1-natechancellor@gmail.com/

**Comment by nathanchance** [comment]
Took a few tries but accepted: https://git.kernel.org/pub/scm/linux/kernel/git/davem/net-next.git/commit/?id=8a962c4aa1e6c0836985a5843f44d74fe3cced12

**Comment by nathanchance** [comment]
Fixed as of next-20111119: https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=8a962c4aa1e6c0836985a5843f44d74fe3cced12
